scripts/clear-cloudflare-cache.js

```diff
@@ -15,9 +15,9 @@
 // - CLOUDFLARE_ZONE_ID: The zone ID to purge. You can find it in the
 // sidebar of the "overview" tab for dvc.org
 
-const fetch = require('isomorphic-fetch');
```

Comment:
why this style change?

Reply:
It was automatic, I assume something was changed.

Reply:
nothing I am aware of at least ... I think we use `;` in the blog

Reply:
I don't see any semicolons in this repository.

Reply:
my bad ... really sorry ... still though we are doing this with blog 🤦‍♂